<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Records</class>
 <widget class="QWidget" name="Records">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рекорды</string>
  </property>
  <widget class="QTableWidget" name="RecordTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>99</y>
     <width>450</width>
     <height>302</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Очки</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Режим</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Кол-во попыток</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сложность</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>10 самых лучших результатов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeModeButton">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>54</y>
     <width>147</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сменить режим</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentModeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>49</y>
     <width>450</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Выбранный режим: mode_name</string>
   </property>
  </widget>
  <zorder>CurrentModeLabel</zorder>
  <zorder>RecordTable</zorder>
  <zorder>label</zorder>
  <zorder>ChangeModeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
